stuff/pipelines/contest_callback.py

```diff
@@ -55,13 +58,9 @@ def __call__(self,
         response = json.loads(body)
         data = ContestCertificate(**response)
 
-        scoreboard = self.client.contest.scoreboard(
-            contest_alias=data.alias,
-            token=data.scoreboard_url)
-        ranking = scoreboard.ranking
         certificates: List[Certificate] = []
 
-        for user in ranking:
+        for user in data.ranking:
```

Comment:
Acá ya podrías almacenar los datos del ranking de usuario en la tupla:
```suggestion
        for user_ranking in data.ranking:
            user = Ranking(**user_ranking)
```